### ISSUE TITLE
Generating ID/title for generic files

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/LegacyMan#416]
We are currently using `Test Title` as the title for DITA documents.

The actual title can be found by using the contents of `<div id="Title" ....><h2>XXX</h2></div>`

**Comment by robintw** [comment]
What should we do about getting the overall title for a page when the page has multiple `<div id="Title">` divs?

For example `Phase_F.html` has:

```
<div id="Title" style="position: absolute; visibility: visible; left: 406px; top: -12px; width: 514px">
            <h2>PHASE F UNITS OVERVIEW B</h2>
</div>
```

and

```
<div id="Title" style="position: absolute; visibility: visible; left: 406px; top: -12px; width: 514px">
            <h2>PHASE F UNITS OVERVIEW B</h2>
        </div>
```
For the titles in the two separate 'sub pages' that we have in that page.

Is there even such a thing as a title for the whole HTML page?

**Comment by IanMayo** [comment]
From examining the data, the first will be called "Title" and the second "Title2", though I wouldn't rely on this.  So, I'm happy that we use the first matching div.

**Comment by robintw** [comment]
Ok - yes we can use the first Title div for the overall time.

What should we do with the second title - should we include it as another title/subtitle somewhere further down the document?

(And Phase_F.html doesn't have Title and Title2, they're both `Title` as far as I can see)

**Comment by IanMayo** [comment]
We can just take the first title for now.  But, we _may_ wish to use supplemental titles as title for other sections in the future, but not now.

**Comment by robintw** [comment]
Fixed in #421
